docker/gateway/Dockerfile-noble

```diff
@@ -13,31 +13,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-FROM graviteeio/java:17-alpine-3.20
+
+FROM eclipse-temurin:17.0.13_11-jre-noble
 LABEL maintainer="contact@graviteesource.com"
 
 ARG GRAVITEEAM_VERSION=0
 ENV GRAVITEEIO_HOME /opt/graviteeio-am-gateway
 
-RUN addgroup -g 1000 graviteeio \
-    && adduser -D -H -u 1001 graviteeio --ingroup graviteeio \
+RUN addgroup -gid 1001 graviteeio && \
+ adduser --disabled-password --no-create-home -uid 1001 graviteeio --ingroup graviteeio
 
-RUN apk update \
-	&& apk add --update --no-cache zip unzip netcat-openbsd wget \
-	&& wget --no-check-certificate -O /tmp/gravitee-am-gateway-standalone-${GRAVITEEAM_VERSION}.zip https://oss.sonatype.org/service/local/artifact/maven/content\?r\=snapshots\&g\=io.gravitee.am.gateway.standalone\&a\=gravitee-am-gateway-standalone-distribution-zip\&p\=zip\&v\=${GRAVITEEAM_VERSION} \
+RUN apt-get update \
+    && apt-get --yes upgrade \
+    && apt-get --yes install zip unzip netcat-openbsd wget \
+	&& wget --no-check-certificate -O /tmp/gravitee-am-gateway-standalone-${GRAVITEEAM_VERSION}.zip https://download.gravitee.io/graviteeio-am/components/gravitee-am-gateway/gravitee-am-gateway-standalone-${GRAVITEEAM_VERSION}.zip \
     && unzip /tmp/gravitee-am-gateway-standalone-${GRAVITEEAM_VERSION}.zip -d /tmp/ \
-	&& apk del zip unzip netcat-openbsd wget \
+    && apt-get --yes remove zip unzip netcat-openbsd wget \
+    && apt autoremove --yes \
     && mv /tmp/gravitee-am-gateway-standalone-${GRAVITEEAM_VERSION} ${GRAVITEEIO_HOME} \
     && rm -rf /tmp/* \
```

Comment:
Why do we apply a manual cleanup here, where on other Dockerfile we use a multistage build to first download and unzip, and then use the binaries ?